tests/test_myplex.py

```diff
@@ -126,6 +126,21 @@ def enabled():
     utils.wait_until(lambda: enabled() == (False, False))
 
 
+def test_myplex_onlineMediaSources_optOut(account):
+    mediaOptOut = account.onlineMediaSources()[0]
```

Comment:
`account.onlineMediaSources()` is empty in the CI test run against an unclaimed server.

Reply:
So I should add these as exempt or skip testing?

Reply:
skipped